README.md

```diff
@@ -90,6 +90,12 @@ By default, only messages with `type: "warning"` warnings are logged. (These are
 
 For example, `function(message) { return true }` will only every message, regardless of the plugin or whether it's a warning or not.
 
+**clearAllMessages** (function)
```

Comment:
If you're still passing a *function* as argument, the previous name `clearMessageFilter`, made more sense. When I proposed `clearAllMessages`, I was thinking it would be just a boolean. Is there a use-case that makes the function more compelling than a boolean?